src/app/screens/Onboard/NewWallet/index.tsx

```diff
@@ -130,10 +138,27 @@ export default function NewWallet() {
           ) : (
             <div className="w-4/5">
               <div className="mt-6 dark:text-white">
-                <strong>Remember, not your keys, not your coins. </strong>
+                <strong>We host a lightning wallet for you!</strong>
                 <br />
-                This quick setup uses a custodial service <br /> to manage your
-                wallet.
+                ...but remember: not your keys, not your coins.
+              </div>
+
+              <div className="mt-6">
+                <label
+                  htmlFor="password"
```

Comment:
Should be "email".

Reply:
To support label-click/functionality do you also need the `id` on the input?

Reply:
@escapedcat correct!

Reply:
uuuupppsss.. failed at prorgamming rule no.1: never c&p 
thanks